playbooks/common/openshift-cluster/std_include.yml

```diff
@@ -7,6 +7,10 @@
   tags:
   - always
 
+- include: sanity_checks.yml
```

Comment:
@mtnbikenc, this feels like the right place but I want to ensure Russ has some input as we are guarding the `std_include.yml` so that it does not expand without a double :+1:  on it.

Reply:
This could be fine for now.  Moving these checks up higher is good, and since these are just `fail` checks they shouldn't add discernable time.